users/engler/fairseq/training.py

```diff
@@ -0,0 +1,97 @@
+import os
+import subprocess as sp
+import yaml
+
+from sisyphus import *
+
+import recipe.i6_core.util as util
+
+
+class DictToYamlJob(Job):
+    """
+    Writes a dict into a .yaml file
+    """
+
+    def __init__(self, config_dict, hash_reset, *, yaml_prefix=""):
+        """
+        :param dict config_dict:
+        :param str hash_reset: Temporary way to reset the sisyphus hash for debugging purposes
+        :param str yaml_prefix: Prefix which should be written to the beginning, for example "# @package _group_" or "#!rnn.py"
+        """
+        assert isinstance(config_dict, dict)
+
+        self.config_dict = config_dict
+        self.yaml_prefix = yaml_prefix
+        self.out_config_file = self.output_path("fairseq_config.yaml")
+        self.file_out = self.output_path("out")
```

Comment:
```suggestion
        self.out_file = self.output_path("out")
```
See https://github.com/rwth-i6/i6_core/blob/main/CONTRIBUTUNG.md

But this doesn't seem to be used anyway, right?

Reply:
I have misinterpreted the guidelines^^. After looking at the `out_*` again, I understand it now
2. In a Sisyphus job class the output variables **end** in `out_*`